WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -58,6 +58,8 @@ def psych_sheet_event
   def index
     @competition = competition_from_params
     @registrations = @competition.registrations.accepted.includes(:user, :events).order("users.name")
+    @count_by_event = Hash.new(0)
+    @newcomers = @returners = 0
```

Comment:
IMO, I don't think there's any value in putting these here as opposed to local variables inside of the view.